### ISSUE TITLE
use `kodiak: merge.method = 'merge'` as automerge-label doesn't work

### ISSUE THREAD

**Issue by clxmstaab** [chdsbd/kodiak#534]
... with this label...

![image](https://user-images.githubusercontent.com/47448731/96243225-b58e7f00-0fa4-11eb-9672-5f2bad708f96.png)

...applied to a PR ...

![image](https://user-images.githubusercontent.com/47448731/96243097-8ed04880-0fa4-11eb-9239-f279332aff2b.png)

...of a project with the following config ...

```
# DONT EDIT THIS FILE OUTSIDE OF complex-gmbh/php-automation-templates
# changes will be one-way synced from the automation repos

# .kodiak.toml
version = 1

[merge]
method = "squash"
# once a PR is merged into master, delete the branch
delete_branch_on_merge = true
# label to use to enable Kodiak to merge a PR
automerge_label = ["automerge", "kodiak: merge.method = 'merge'"]
# require that the automerge label be set for Kodiak to merge a PR. if you
# disable this Kodiak will immediately attempt to merge every PR you create
require_automerge_label = true
# if a PR is passing all checks and is able to be merged, merge it without
# placing it in the queue. This will introduce some unfairness where those
# waiting in the queue the longest will not be served first.
prioritize_ready_to_merge = true
# Kodiak will not merge a PR with any of these labels.
blacklist_labels = ["misses-clxLBE"]

[merge.message]
title = "pull_request_title"
body = "pull_request_body"
include_pr_number = true
strip_html_comments = true # remove html comments to auto remove PR templates

[update]
# update PR whenever the PR is out of date with the base branch. PR will be
# updated regardless of failing status checks, missing reviews, merge blacklist
# labels, etc. Kodiak will only update the PR if the automerge label is attached
# or `update.require_automerge_label` is false.
always = true
require_automerge_label = true
```

I would expect kodiak to merge the PR with a merge commit.

kodiak reports that my PR is missing the automerge-label though.

after we additionally add the `automerge` label it merges like expected (but as discussed in https://github.com/chdsbd/kodiak/issues/476#issuecomment-701988962 we would love to only need a single label)

**Comment by chdsbd** [comment]
I think this may have been a problem where Kodiak missed an event for this PR. I tested your config and I was able to merge a pull request via `kodiak: merge.method = 'merge'`.

As a workaround you can update the pull request (edit the description or title, add/remove a label, etc.) and that should trigger Kodiak to reevaluate the pull request.

Out of curiosity, if you click "Details" to the right of the Kodiak status check, was that status check set after the approving review?

**Comment by clxmstaab** [comment]
ok thx.. I will check whether it happens again.

as you said, we had other cases which worked like expected since I reported the issue.

> Out of curiosity, if you click "Details" to the right of the Kodiak status check, was that status check set after the approving review?

the PR was merged, so this information is no longer available

**Comment by clxmstaab** [comment]
looks good in another example. will close for now and report back in case I can reproduce.

sorry for the noise.
